lib/livebook_web/helpers/html_helpers.ex

```diff
@@ -36,7 +36,15 @@ defmodule LivebookWeb.HTMLHelpers do
     name
     |> String.trim()
     |> String.downcase()
+    |> String.replace(~r/[^\s\w]/u, "")
+    # We need to trim again after removing special characters
+    # in case of the last one was removed from the above regex
+    |> String.trim()
```

Comment:
I think the later trim is enough :D

```suggestion
    |> String.downcase()
    |> String.replace(~r/[^\s\w]/u, "")
    |> String.trim()
```